src/classes/modules/java.base/java/util/ServiceLoader.java

```diff
@@ -0,0 +1,132 @@
+package java.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.InvocationTargetException;
+import java.net.URL;
+import java.util.function.Supplier;
+
+public class ServiceLoader<S> implements Iterable<S> {
```

Comment:
Add a Javadoc-style comment describing what this class is about.

Reply:
Javadoc-style comment added